src/App.js

```diff
@@ -0,0 +1,31 @@
+import './App.css';
+import React from 'react';
+import Clock from './components/clock/Clock';
+import Weather from './components/weather/Weather';
+import LandingPage from './pages/landingPage/LandingPage';
+import { Route, Routes } from "react-router-dom";
+import MainPage from './pages/mainPage/MainPage';
+import HomePage from './pages/homePage/HomePage';
+import Quote from './components/quoteComponent/Quote';
```

Comment:
suggestion: export all components from index.js in components folder and import from it